recipe/meta.yaml

```diff
@@ -25,7 +25,7 @@ requirements:
     - python >={{ python_min }}
     - sqlite
     - six >=1.16.0
-    - globus-sdk >=3.15.0
+    - globus-sdk >=3.15.0,<4.0
```

Comment:
Without this, we get `AttributeError: 'TransferClient' object has no attribute 'endpoint_autoactivate'`. It should be possible to fix the erroring code itself, but I'm not sure how long that will take to debug. I think at this point in the testing period, we should just pin the versions like this.